docs/how-to-guides/set-global-configs.md

```diff
@@ -5,9 +5,13 @@ This is an experimental feature, and may change at any time. Learn [more](faq.md
 
 Promptflow supports setting global configs to avoid passing the same parameters to each command. The global configs are stored in a yaml file, which is located at `~/.promptflow/pf.yaml` by default.
 
+Additionally, promptflow supports setting configs to a specified path, and these configs will only take effect for the Promptflow program when the working directory is the specified path or its subdirectories.
```

Comment:
consider change the file title to: set prompt flow configs

Reply:
Okay, the name has been changed.